lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, fig, rect=None, *args, **kwargs):
           *elev*             Elevation viewing angle (default 30)
           *zscale*           [%(scale)s]
           *sharez*           Other axes to share z-limits with
+          *persp*            Perspective projection (default True)
```

Comment:
I agree. In that case, should I provide the setter `set_proj`? There is a `get_proj` which has different meaning.